test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -35,13 +35,53 @@ internal static class RegressionTest
 
         private static int Main(string[] args)
         {
-            return Parser.Default.ParseArguments<Options>(args).MapResult(
-                Run,
-                _ =>
+            if (args.Length >= 1 && args[0].Equals("warm-up"))
```

Comment:
We could have a `--warm-up` command line option, and call this executable many times in pipeline definition.

Reply:
After exploring, I think the current solution is better to download all repos to every agents. A `--warm-up` command option seems to make the pipeline complex.